cocotb/handle.py

```diff
@@ -130,7 +130,7 @@ def __str__(self):
     def __setattr__(self, name, value):
         if name in self._compat_mapping:
             if name not in _deprecation_warned:
-                warnings.warn("Use of %s attribute is deprecated" % name)
+                warnings.warn("Use of attribute %r is deprecated, use %r instead" % (name, self._compat_mapping[name]))
                 _deprecation_warned[name] = True
```

Comment:
Out of scope for this patch, but this should really be
```
warnings.warn("...", DeprecationWarning, stacklevel=2)
```
(leaving python to deal with the "already-warned" state)

We'll want to fix #1194 before we make that change.